src/black/comments.py

```diff
@@ -157,9 +160,18 @@ def make_comment(content: str) -> str:
     if (
         content
         and content[0] == "\N{NO-BREAK SPACE}"
-        and not content.lstrip().startswith("type:")
+        and not is_type_comment_string("# " + content.lstrip(), mode=mode)
     ):
         content = " " + content[1:]  # Replace NBSP by a simple space
+    if (
+        Preview.standardize_type_comments in mode
+        and content
+        and "\N{NO-BREAK SPACE}" not in content
+        and is_type_comment_string("# " + content.lstrip(), mode=mode)
```

Comment:
```suggestion
        and is_type_comment_string("#" + content, mode=mode)
```

Just for the sake of conciseness - `.lstrip()` is already run within `is_type_comment_string()`

I think it makes sense to keep it as-is on L163, since there we might still be on stable style and can't rely on `is_type_comment_string()` to strip it.